src/foam/u2/crunch/EasyCrunchWizard.js

```diff
@@ -16,8 +16,7 @@ foam.CLASS({
   properties: [
     {
       class: 'Boolean',
-      name: 'allowSkipping',
-      value: true
```

Comment:
The default wasn't true before? - or is this overriding the default?

Either way `allowSkipping: false` is a much more reasonable default IMO so I'm approving this.